requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
+boto==2.45.0
```

Comment:
The thing is that right now the Dockerfile uses playbook2image's `assemble` script to put things together, including python package install via `pip install -r requirements.txt`. In other words: I think that the way to install Boto into the image in a "nice" way that fits how things are set up now is to add it to `requirements.txt`.

I didn't think of a negative impact here (unlike the Dockerfile.rhel7 case)... how bad is it to have that non-hard requirement in `requirements.txt`?